scipy/spatial/_procrustes.py

```diff
@@ -38,21 +38,20 @@ def procrustes(data1, data2):
     Parameters
     ----------
     data1 : array_like
-        matrix, n rows represent points in k (columns) space data1 is the
+        Matrix, n rows represent points in k (columns) space data1 is the
         reference data, after it is standardised, the data from data2 will
         be transformed to fit the pattern in data1 (must have >1 unique
         points).
-
     data2 : array_like
-        n rows of data in k space to be fit to data1.  Must be the  same
-        shape (numrows, numcols) as data1 (must have >1 unique points).
+        n rows of data in k space to be fit to `data1`.  Must be the  same
```

Comment:
I think the backticks should be removed from this "data1" or backticks should be added for the other occurrences of data1 and data2 in the description of the parameters. Not a problem really, just an observation.